src/LodeRunner/src/Services/LodeRunnerService.cs

```diff
@@ -196,6 +201,41 @@ public void LogStatusChange(object sender, ClientStatusEventArgs args)
             Console.WriteLine($"{args.Message} - {args.LastUpdated:yyyy'-'MM'-'dd'T'HH':'mm':'ss.fffffffK}"); // TODO fix LogStatusChange implementation
         }
 
+        /// <summary>
+        /// Updates the TestRun with LoadResults.
+        /// </summary>
+        /// <param name="sender">The sender.</param>
+        /// <param name="args">The <see cref="LoadResultEventArgs"/> instance containing the event data.</param>
+        public async void UpdateTestRun(object sender, LoadResultEventArgs args)
```

Comment:
We recently introduced ClientStatusUpdater (under Subscriber folder) look like UpdateTestRun logic could be refactor into a separate Subscriber class like ClientStatusUpdater.

Reply:
I think this could be a separate task.

Reply:
Agreed this should be a separate task.  This is what I was calling out in a comment above (I realize now I should have been more specific).